### PR TITLE
enable tiiuae/falcon-11B-vlm in image_to_text example. fix the incorr…

### PR DESCRIPTION
**Pull request by sywangyi**
…ect output

# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by sywangyi** [comment]
|  throught(tokens/s)| A100 | Gaudi2 |
|-------|------|----------|
| BF16 | 14.37 |  25.84 |

command:
Gaudi2 python3 run_pipeline.py --model_name_or_path tiiuae/falcon-11B-vlm  --bf16   --max_new_tokens 100 --use_hpu_graphs

A100: python3 run_pipeline.py --model_name_or_path tiiuae/falcon-11B-vlm  --bf16   --max_new_tokens 100

File: optimum/habana/transformers/models/clip/modeling_clip.py [hunk 1]
```diff
@@ -102,9 +104,17 @@ def forward(
...
                 attn_output = self.fused_scaled_dot_product_attention(
-                    query_states, key_states, value_states, attention_mask, self.dropout, False, 1, "fast"
+                    query_states,
+                    key_states,
+                    value_states,
+                    attention_mask,
+                    self.dropout,
+                    False,
+                    1,
+                    softmax_mode,
```
[the remark this answers is not in the document]
Comment (sywangyi) [reply optimum/habana/transformers/models/clip/modeling_clip.py:117]: I check the case under test. there's no case with use_flash_attention=True for clip model.

[the remark this answers is not in the document]
Comment (sywangyi) [reply optimum/habana/transformers/models/clip/modeling_clip.py:117]: also for llava which use clip as image model. it run with use_flash_attention=False

**Comment by yeonsily** [comment]
LGTM

**Review by yeonsily — APPROVED** [review]

**Review by regisss — APPROVED** [review]

**Pull request merged**